### PR TITLE
Amls 4756

### PR DESCRIPTION
**Pull request by ajchilts**
1). change About the business to Business details on the progress page (application progress, renewal progress, update or view your information)

2). change the sub heading in the About the business section from About the business to Business details

This change brings the back end models in line with the front end changes for this ticket. 

## Related / Dependant PRs?

https://github.com/hmrc/amls-frontend/pull/973
https://github.com/hmrc/amls-acceptance-test/pull/151

## How Has This Been Tested?

Unit testing, manual testing and regression pack.

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [ ] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

**Comment by ajchilts** [comment]
Screenshot 2019-03-15 at 12 51 25

File: app/models/des/aboutthebusiness/BusinessContactDetails.scala [hunk 1]
```diff
@@ -29,7 +29,7 @@ case class BusinessContactDetails (businessAddress: Address,
 object BusinessContactDetails {
   implicit val format = Json.format[BusinessContactDetails]
 
-  implicit def convert(aboutTheBusiness: AboutTheBusiness) : BusinessContactDetails = {
+  implicit def convert(aboutTheBusiness: BusinessDetails) : BusinessContactDetails = {
```
Comment (slavvco) [inline app/models/des/aboutthebusiness/BusinessContactDetails.scala]: there's aboutTheBusiness name left in convert function above (and below), any reason behind that?

File: app/models/fe/SubscriptionView.scala [hunk 2]
```diff
@@ -37,7 +37,7 @@ case class SubscriptionView(
                              businessMatchingSection: BusinessMatching,
                              eabSection: Option[EstateAgentBusiness],
                              tradingPremisesSection: Option[Seq[TradingPremises]],
-                             aboutTheBusinessSection: AboutTheBusiness,
+                             aboutTheBusinessSection: BusinessDetails,
```
Comment (slavvco) [inline app/models/fe/SubscriptionView.scala]: like above

File: app/models/fe/SubscriptionRequest.scala [hunk 3]
```diff
@@ -35,7 +35,7 @@ case class SubscriptionRequest(
                                 businessMatchingSection: BusinessMatching,
                                 eabSection: Option[EstateAgentBusiness],
                                 tradingPremisesSection: Option[Seq[TradingPremises]],
-                                aboutTheBusinessSection: AboutTheBusiness,
+                                aboutTheBusinessSection: BusinessDetails,
```
Comment (slavvco) [inline app/models/fe/SubscriptionRequest.scala]: like above

File: app/models/des/aboutthebusiness/VATRegistration.scala [hunk 4]
```diff
@@ -25,7 +25,7 @@ object VATRegistration {
 
   implicit val format = Json.format[VATRegistration]
 
-  implicit def convert(aboutTheBusiness: AboutTheBusiness): Option[VATRegistration] = {
+  implicit def convert(aboutTheBusiness: BusinessDetails): Option[VATRegistration] = {
```
Comment (slavvco) [inline app/models/des/aboutthebusiness/VATRegistration.scala]: like above

File: app/models/des/aboutthebusiness/PreviouslyRegisteredMLRView.scala [hunk 5]
```diff
@@ -27,7 +27,7 @@ case class PreviouslyRegisteredMLRView(amlsRegistered:Boolean,
 object PreviouslyRegisteredMLRView{
   implicit val format = Json.format[PreviouslyRegisteredMLRView]
 
-  implicit def convert(aboutTheBusiness:AboutTheBusiness):Option[PreviouslyRegisteredMLRView] ={
+  implicit def convert(aboutTheBusiness:BusinessDetails):Option[PreviouslyRegisteredMLRView] ={
```
Comment (slavvco) [inline app/models/des/aboutthebusiness/PreviouslyRegisteredMLRView.scala]: like above

File: app/models/des/aboutthebusiness/PreviouslyRegisteredMLR.scala [hunk 6]
```diff
@@ -28,7 +28,7 @@ case class PreviouslyRegisteredMLR(amlsRegistered:Boolean,
 object PreviouslyRegisteredMLR{
   implicit val format = Json.format[PreviouslyRegisteredMLR]
 
-  implicit def convert(aboutTheBusiness:AboutTheBusiness):Option[PreviouslyRegisteredMLR] ={
+  implicit def convert(aboutTheBusiness:BusinessDetails):Option[PreviouslyRegisteredMLR] ={
```
Comment (slavvco) [inline app/models/des/aboutthebusiness/PreviouslyRegisteredMLR.scala]: like above

**Pull request merged**